software/findKeyboards.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python
+
+# finds keyboards connected to system
+import evdev
+
+def main():
+	print("Input devices connected to machine:")
+	devices = [evdev.InputDevice(x) for x in evdev.list_devices()]
+	for i in devices:
+		print devices[devices.index(i)]
+
+if __name__ == "__main__":
+	main()
```

Comment:
Add trailing newline, fix indentation to be reasonable (like, 4 spaces per PEP8 or 2 spaces per gpystyle)